### ISSUE TITLE
add bodhran music notation

### ISSUE THREAD

**Issue by Reaper10** [canorusmusic/canorus#185]
can you add support for the irsh bodhran music notation

**Comment by suamor** [comment]
I do not now anything about bodhran notation.
I did a search but did not come up with anything about this kind of notation.
Lilypond documentation does not seem to know this notation under this name.

Could you point us to example and what is missing in Canorus?

We are largely relying on lilypond. If lilypond is missing notation parts required for bodhran notation, it needs to be implemented in lilypond before we can add it to Canorus,

Generally I suggest to add this notation type as a plugin.
